docs/admin/authorization/rbac.md

```diff
@@ -693,11 +693,11 @@ subjects:
   name: user-1
 ```
 
-When bootstrapping the first roles and role bindings, it is necessary for the initial user to grant permissions they do not yet have.
-To bootstrap initial roles and role bindings:
-
-* Use a credential with the `system:masters` group, which is bound to the `cluster-admin` super-user role by the default bindings.
-* If your API server runs with the insecure port enabled (`--insecure-port`), you can also make API calls via that port, which does not enforce authentication or authorization.
+When bootstrapping the first roles and role bindings, it is necessary for the
+initial user to grant permissions they do not yet have.
```

Comment:
As long as we're editing, let's clean up this text too (ll696-697):
When you bootstrap the first roles and role bindings, you must grant permissions that you do not yet have.

(I'm not completely sure this is technically correct, but the original is not clear either. Is this work for the admin or for the user/operator? Who is this "initial user"? The admin? Operator? The same person who bootstraps, or someone else? I'm pretty sure I know the answers bc I know RBAC pretty well. But the English makes me wonder ...)